tools/tf2openapi/cmd/main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"flag"
+	"github.com/golang/protobuf/proto"
+	"github.com/kubeflow/kfserving/tools/tf2openapi/generator"
+	pb "github.com/kubeflow/kfserving/tools/tf2openapi/generated/protobuf"
+	"io/ioutil"
+	"log"
+	"os"
+)
+
+var (
+	model = flag.String("model", "", "Absolute path of SavedModel file")
+	out   = flag.String("out", "std", "Either 'std' to display on standard out or 'file' to save output")
+	// outFile not required if output option is std
+	outFile = flag.String("outFile", "", "Absolute path of file to write OpenAPI spec to")
```

Comment:
Don't use camel for flags. Typically id expect to do something like --out-file=

Reply:
Going with --output-file!